tests/test_connection_manager.c

```diff
@@ -161,13 +161,11 @@ static int s_cm_tester_init(struct cm_tester_options *options) {
         clock_fn = options->mock_table->aws_high_res_clock_get_ticks;
     }
 
-    tester->event_loop_group = aws_event_loop_group_new(
-        tester->allocator,
-        clock_fn,
-        1,
-        NULL,  // new_loop_fn
-        NULL,  // new_loop_user_data
-        NULL); // shutdown_options
+    struct aws_event_loop_group_options elg_options = {
```

Comment:
ah, is this because I had a newer aws-c-io version locally that I was running against? That makes sense.

Reply:
Yes, sorry, I meant the other way around. I was indeed on an older version.